util/bulk-transfer.py

```diff
@@ -0,0 +1,79 @@
+import os
+import json
+from neo.contrib.narrative.bulk_process_tx import BulkProcess
+
+
+class BulkTransfer(BulkProcess):
+
+    from_address = None
+
+    completed_jobs_path = None
+    completed_jobs = None
+    job_key = None
+
+    def __init__(self):
+        with open(os.path.join(os.path.abspath(os.path.dirname(__file__)), 'config', 'bulk-transfer-config.json'), 'r') as f:
```

Comment:
I had to look this up to understand it. So path is the OS's version of a path separator, and it is joining everything inside, which is then opened as a read only file stream... Okay... So you don't want a leading slash because the assumption is that the config folder is in the same directory as the script itself?

Reply:
It'll give the absolute path of the file specified. `__file__` is the current file, so it basically gives you a full path to the script's directory, and then we reference the config file from there.

FWIW, this is just boilerplate copy/paste from the other classes that do the same thing. Ultimately, I mirrored this off of neo-python's original sample templates...